pkg/config/amqStreams.go

```diff
@@ -1,8 +1,8 @@
 package config
 
 import (
+	kafkav1 "github.com/integr8ly/integreatly-operator/pkg/apis-products/kafka.strimzi.io/v1alpha1"
```

Comment:
```suggestion
	kafkav1alpha1 "github.com/integr8ly/integreatly-operator/pkg/apis-products/kafka.strimzi.io/v1alpha1"
```

Reply:
Good spotting, resolved now and for the rest